### PR TITLE
moving content out of sequence folder to steps

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-489

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/common/common.content.ts [hunk 1]
```diff
@@ -123,8 +121,6 @@ const cy: typeof en = {
   wife: 'gwraig',
   partner: 'partner',
   civilPartner: 'partner sifil',
-  marriage: 'priodas',
-  civilPartnership: 'partneriaeth sifil',
```
Comment (SebFlippence) [inline src/main/steps/common/common.content.ts:127]: Did you mean to delete these translations?

Comment (velimd) [reply src/main/steps/common/common.content.ts:127]: the property already exist

Comment (velimd) [reply src/main/steps/common/common.content.ts:127]: [just a couple of lines above](https://github.com/hmcts/nfdiv-frontend/pull/296/files/8893afce7842c3cd874067fc250177090c333078#diff-fdd46b7ae6a19351332967acc233ae0022097bfd3f13fd879a434f2c52768617R118)

Comment (SebFlippence) [reply src/main/steps/common/common.content.ts:127]: As yes, must have come from 2 very similar PRs with the same/similar translations 👍

**Comment by SebFlippence** [comment]
Looks like there's a conflict with `src/main/steps/sequence/your-details/content.ts`

**Pull request closed without merging**